form/index.js

```diff
@@ -81,13 +80,30 @@ class TonicForm extends Tonic {
     this.state = data
   }
 
+  relay (name) {
+    this.dispatchEvent(new window.CustomEvent(name, { bubbles: true }))
+  }
+
+  input (e) {
+    this.change(e)
+  }
+
+  change (e) {
+    const el = Tonic.match(e.target, '[data-key]')
+    if (!el) return
+
+    this.relay(e.type)
```

Comment:
does this double emit `input` & `change` ? these events bubble by default.

Reply:
In the other components we relay `focus` & `blur` because those events do not bubble by default.

Reply:
oh yeah, good catch, I cant remember why I added this 💫